docs/reference/api/drivers/RawSerial.md

```diff
@@ -1 +1,34 @@
 ## RawSerial
+
+The RawSerial class provides UART functionality without the use of streams like the Serial class. This makes it suitable for use in interrupt handlers with the RTOS.
```

Comment:
stream is out of context (and should be Stream - maybe with a link??). It doesn't re-target against the standard library. Probably better to say it partially re-implements scan and print family functions (need to fact check how true this is)

Reply:
Serial calls into Stream's printf which calls into vfprintf to print through an output stream. RawSerial re-implements printf using vsprintf to write the formatted string to a buffer and writes to serial using _base_putc() -> target's serial_putc definition so looks like it checks out. 

Updated the phrasing and added a link to the Stream class.